document/static/js/modules/schema/document.js

```diff
@@ -407,15 +407,55 @@ spec.nodes = spec.nodes.update(
         spec.nodes.get("table"),
         {
             attrs: {
-                track: {default: []}
+                track: {default: []},
+                width:{default:"100"},
+                aligned:{default:'center'}
             },
             parseDOM: [{tag: "table", getAttrs(dom) {
                 return {
-                    track: parseTracks(dom.dataset.track)
+                    track: parseTracks(dom.dataset.track),
+                    width: dom.width,
+                    aligned: dom.aligned
                 }
             }}],
             toDOM(node) {
-                const attrs = {}
+                const attrs ={}
+                switch(node.attrs.width){
+                    case "25":
```

Comment:
Why make this long switch? It seems like something like this should cover it:

```
attrs['data-width'] = node.attrs.width
attrs['class'] = `table-${node.attrs.width}`

Reply:
Fixed in new commit